packages/app/src/containers/Inspector.tsx

```diff
@@ -52,6 +53,7 @@ export default memo(function Inspector() {
           };
 
     const hasTextLayer = selectedLayers.some((l) => l._class === 'text');
+    const hasAllTextLayer = selectedLayers.every((l) => l._class === 'text');
```

Comment:
I recently added a `Layers.isTextLayer` the we can use instead of the `_class` comparison. Not super important, but looks a little clearer I think